isaric/parsers/isaric-ecmo.toml

```diff
@@ -1116,6 +1121,18 @@
   date = { ref = "dailyDateHierarchy" }
   value = { field = "daily_temp_vsorres", unit = "°C", source_unit = { field = "daily_temp_vsorresu", values = { 1 = "°C", 2 = "°F" } } }
 
+[[observation]]
+  name = "fio2_percent"
+  phase = "study"
+  date = { ref = "dailyDateHierarchy" }
+  value = { field = "daily_fio2b_lborres", apply = { function = "Percentage" } }
```

Comment:
what is the difference between fio2 and fio2b?

Reply:
fio2b contains values as a percentage, fio2 as decimals (or possible vv.) - but both appear to contain a mix of decimals and percentage values in practice

Reply:
can we create a function to convert to percentage only if the value is between 0.21 and 1.0? and leave as is if between 21 and 100 (%). There are probably values in between 1 and 21 that will have to remain as reported. Would that be possible, as I don't think there's ambiguity in the units if the values are within those clearly defined ranges